ui/app/templates/components/clients/running-total.hbs

```diff
@@ -54,21 +49,17 @@
     </div>
 
     <div class="data-details-top">
-      {{#let (get this.getAverageNewClients 0) as |stat|}}
-        <h3 class="data-details">Average new entity clients per month</h3>
-        <p class="data-details">
-          {{format-number stat.average}}
-        </p>
-      {{/let}}
+      <h3 class="data-details">Average new entity clients per month</h3>
+      <p class="data-details">
+        {{format-number (get this.getEntityClientData "averageNewClients")}}
+      </p>
     </div>
 
     <div class="data-details-bottom">
-      {{#let (get this.getAverageNewClients 1) as |stat|}}
-        <h3 class="data-details">Average new non-entity clients per month</h3>
-        <p class="data-details">
-          {{format-number stat.average}}
-        </p>
-      {{/let}}
+      <h3 class="data-details">Average new non-entity clients per month</h3>
+      <p class="data-details">
+        {{format-number (get this.getNonEntityClientData "averageNewClients")}}
```

Comment:
The instances in this file that use the `get` helper are not necessary since you know the name of the property you are trying to access. Instead you can replace it with `this.getNonEntityClientData.averageNewClients`.

Reply:
Thanks! I tried that earlier and it didn't work - which is what prompted me reaching out in slack. Might have been some other syntax or typo issue as it works now! 🤯